index.js

```diff
@@ -36,9 +38,11 @@ module.exports = (api, options) => {
 
   const cordovaRun = () => {
     // cordova run platform
-    info('executing "cordova run ' + platform + '"...')
+    // uses build for browser since run would trigger cordovaServe
+    const cmd = platform === 'browser' ? 'build' : 'run'
```

Comment:
This kept me stuck for a while. Looks like the cordova browser platform run triggers a server, see [here](https://github.com/apache/cordova-browser/blob/8847dd878af227e29901495cc70dcd1090343380/bin/template/cordova/lib/run.js#L48). 

To get around this limitation, and since there is no arg to avoid the server to be launched it call `build` instead.